tensorflow/compiler/xla/service/allocation_tracker.h

```diff
@@ -63,10 +63,10 @@ class Allocation {
     CHECK_GE(ref_count_, 0);
     return ref_count_;
   }
-  void increment_ref_count() {
+  void increment_ref_count(int inc) {
     CHECK_GT(ref_count_, 0);
-    CHECK_LT(ref_count_, INT_MAX);
-    ++ref_count_;
+    CHECK_GT(ref_count_+inc, 0);
```

Comment:
ok.  i was assuming that an overflow would result in a negative number. however your solution is nicer.  in practice i wouldn't expect a refcount to get anywhere close to MAX_INT